Documentation/openshift.md

```diff
@@ -0,0 +1,101 @@
+---
+title: OpenShift
+weight: 17
+indent: true
+---
+
+# OpenShift
+
+[OpenShift](https://www.openshift.com/) adds a number of security and other enhancements to Kubernetes. In particular, [security context constraints](https://blog.openshift.com/understanding-service-accounts-sccs/) allow the cluster admin to define exactly which permissions are allowed to pods running in the cluster. You will need to define those permissions that allow the Rook pods to run. 
+
+## Rook Privileges
+
+To orchestrate the storage platform, Rook requires the following access in the cluster:
+- Create `hostPath` volumes, for persistence by the Ceph mon and osd pods
+- Run pods in `privileged` mode, for access to `/dev` and `hostPath` volumes
+- Host networking
+
+## Security Context Constraints
+
+Before starting the Rook operator or cluster, create the security context constraints needed by the Rook pods. The following yaml is found in `rook-scc.yaml` under `/cluster/examples/kubernetes`.
+
+```yaml
+kind: SecurityContextConstraints
+apiVersion: v1
+metadata:
+  name: rook
+allowPrivilegedContainer: true
+allowHostNetwork: true
```

Comment:
as @jbw976 pointed out, the [agent](https://github.com/rook/rook/blob/master/pkg/operator/ceph/agent/agent.go#L149) needs `hostNetwork: true` because of udev notifications by the rbd tool, so it looks like we need to require it.

Reply:
argh. another thing I'm hoping local volumes will fix.